rfd/00XX-ci.md

```diff
@@ -0,0 +1,83 @@
+
+---
+author: jane quintero (jane@goteleport.com) Russell Jones (rjones@goteleport.com)
+state: draft
+---
+
+# Continuous Integration (CI)
+
+## What/Why
+
+This RFD proposes changes to Continuous Integration (CI) at Teleport to increase security, reduce complexity, and build a base for future enhancements.
+
+This includes, but is not limited to, the following functionality.
+
+* Automating parts of the development process (approvals, assignments) with bots.
+* Running tests (unit, integration) against internal and external PRs.
```

Comment:
Having written this out now, I feel GitHub Actions may not be a good fit for tests. The other things it's still a good fit for, but the lack of manual approval before running a Workflow isn't great.